### PR TITLE
v1.10.4 Changelog

### PR DESCRIPTION
**Pull request by zeripath**
v1.10.4 Changelog on v1.10

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.4](https://github.com/go-gitea/gitea/releases/tag/v1.10.4) - 2020-02-16
+
+* FEATURE
+  * Prevent empty LDAP search from deactivating all users (#9879) (#9890)
+* BUGFIXES
+  * Fix reply on code review (#10261)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Fix reply on code review (#10261) (#10227)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ This changelog goes through all the changes that have been made in each release
...
+
+* FEATURE
+  * Prevent empty LDAP search from deactivating all users (#9879) (#9890)
+* BUGFIXES
+  * Fix reply on code review (#10261) (#10227)
+  * Fix branch page pull request title and link error (#10092) (#10098)
+  * Fix milestone API state parameter unhandled (#10049) (#10053)
+  * Fix wiki raw view on sub path (#10002) (#10041)
+  * Fix RocketChat Webhook (#9908) (#9921) (#9925)
+  * fix bug about wrong dependencies permissions check and other wrong permissions check (#9884)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Fix bug about wrong dependencies permissions check and other wrong permissions check (#9884) (Partial backport #9842)
```

**Review by 6543 — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Comment by sapk** [comment]
CI fail on pgsql (even after multiple restart). Seems the xorm driver is nil before calling .Close

File: .drone.yml [hunk 3]
```diff
@@ -30,6 +30,7 @@ services:
     image: postgres:9.5
     environment:
       POSTGRES_DB: test
+      POSTGRES_HOST_AUTH_METHOD: trust
```
Comment (zeripath) [inline .drone.yml]: This is required due to docker-library/postgres#658

Comment (sapk) [reply .drone.yml]: It seems better to explicitly set POSTGRES_PASSWORD

Comment (zeripath) [reply .drone.yml]:
```suggestion
      POSTGRES_PASSWORD: postgres
```

**Review by guillep2k — APPROVED** [review]

**Pull request merged**